requirements/segmentation.txt

```diff
@@ -6,3 +6,4 @@ scikit-image
 mmdeploy==0.14.0
 timm==0.6.12
 pytorchcv
+einops==0.6.1
```

Comment:
Do we need this for running semisl? Did you check this version no crush with other packages?

Reply:
I don't see any problems with other packages. Yes, unfortunately, this library intensively used by ProtoNetwork. It allows us to compute tensor operations efficiently. It is very popular library by the way:
https://github.com/arogozhnikov/einops